src/sensors/humidity.rs

```diff
@@ -0,0 +1,41 @@
+//! Module providing humidity sensor functionality.
+
+use super::{LocalisedSensorMetadata, SensorTemplate, Sensors};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq)]
+pub struct HumiditySensor {
+    #[serde(flatten)]
+    pub metadata: LocalisedSensorMetadata,
+    pub unit: String,
+    pub value: f64,
+}
+
+#[derive(Debug, Clone)]
+pub struct HumiditySensorTemplate {
+    pub metadata: LocalisedSensorMetadata,
+    pub unit: String,
```

Comment:
Should the `unit` actually be an `enum`? Anyway I wouldn't change it in this PR, since it was a `String` before.